src/olympia/abuse/models.py

```diff
@@ -50,6 +122,12 @@ def send(self):
         send_mail(
             six.text_type(self), msg, recipient_list=(settings.ABUSE_EMAIL,))
 
+    def save(self, *args, **kwargs):
+        creation = not self.pk
+        super(AbuseReport, self).save(*args, **kwargs)
+        if creation:
+            self.send()
```

Comment:
I always wondered if a `post_save` might be cleaner as we already get the information if we are creating or updating and Django can determine that on it's own :thinking:

Reply:
Yeah, I always struggle with this choice. Signals can be hard to follow, so I tend to try to override `save` these days...